Header header
uint16[] tanforce_1
uint16[] normforce_1
uint16[] tanforce_2
uint16[] normforce_2
uint8[] intensity_1
uint8[] intensity_2
